### ISSUE TITLE
Example usage

### ISSUE THREAD

**Issue by barneycarroll** [barneycarroll/asarg#1]
A 1-LOC script npm package mandates more a > 1 line readme

**Comment by tomek-he-him** [comment]
:+1: I’m anxious :)

BTW, after months of avoiding using `this` altogether I’m more and more convinced it was a good idea after all. Ever heard of **[trine](https://github.com/jussi-kalliokoski/trine)**?

**Comment by barneycarroll** [comment]
Yes! My thoughts exactly! `this` has been a nightmare in Javascript, and I've avoided it like the plague because it creates so much confusion with no clear benefits. But then I saw Trine (it's beautiful ) and realised ES7's bind operator can change that, because you can become so much more expressive. Don't get me wrong, I still dislike prototypal inheritance, and I think ES6 `class` is tragic. But with ES7 object property spread and bind we've suddenly got an immense new (haha) context got `this`. I'll try and close this issue tomorrow :)

**Comment by dead-claudia** [comment]
This is fixed in one part of #2, BTW.
